core/transport/src/main/proto/grpc/version.proto

```diff
@@ -20,11 +20,12 @@ enum ServiceType {
   METRICS_MASTER_CLIENT_SERVICE = 9;
   JOB_MASTER_CLIENT_SERVICE = 10;
   JOB_MASTER_WORKER_SERVICE = 11;
-  FILE_SYSTEM_WORKER_WORKER_SERVICE = 12;
+  FILE_SYSTEM_WORKER_WORKER_SERVICE = 12 [deprecated = true];
```

Comment:
seems like it's still used in the code, should we update the usage?

Reply:
I should have updated all the usage. It's still in the generated code for `ServiceType`, but should be nowhere else.

Reply:
What exactly does this `[deprecated = true]` do? What would clients from a older version see when connecting to a new worker after this change?

Reply:
It does little for an enum branch. Just add a `@Depricated` annotation on the generated code. Also `FILE_SYSTEM_WORKER_WORKER_SERVICE` is only used inside alluxio server currently.